tests/test_model_upload.py

```diff
@@ -112,12 +122,15 @@ def test_model_upload_version_with_valid_handle(self):
                     model_upload("metaresearch/llama-2/pyTorch/7b", temp_dir, "Apache 2.0", "model_type")
 
     def test_model_upload_with_too_many_files(self):
+        KaggleAPIHandler.UPLOAD_BLOB_FILE_NAMES = []
```

Comment:
Can you move this to a `TestCase#setUp` method at the top?

https://docs.python.org/3/library/unittest.html#unittest.TestCase.setUp